charts/datadog/templates/kpi-telemetry-configmap.yaml

```diff
@@ -6,6 +6,11 @@ metadata:
   labels:
 {{ include "datadog.labels" . | indent 4 }}
 data:
-  install_id: {{ uuidv4 | quote }}
   install_type: k8s_manual
+  {{- if .Values.datadog.apm.instrumentation.skipKPITelemetry }}
+  install_id: "default"
+  install_time: "default"
```

Comment:
Do you think it is possible to instead set them to `"00000000-0000-0000-0000-000000000000"` and `0`, respectively, so that they are still formatted like a valid UUID and timestamp instead of a string?